recipes/arrow/all/conanfile.py

```diff
@@ -132,6 +141,22 @@ def configure(self):
         if self.options.with_llvm == False and self._with_llvm(True):
             raise ConanInvalidConfiguration("with_openssl options is required (or choose auto)")
 
+        if self.options.with_backtrace:
+            raise ConanInvalidConfiguration("CCI has no backtrace recipe (yet)")
```

Comment:
I think it has `libbacktrace/cci.20210118`

Reply:
@SSE4
Thank you! 
Now this recipe supports libbacktrace.